src/classes/RD_RecurringDonations.cls

```diff
@@ -503,8 +503,8 @@ public with sharing class RD_RecurringDonations {
             }
             
             if (!badSRs.isEmpty()) {
-            	ERR_Handler errorHandler = new ERR_Handler();
-                errorHandler.processInsertResults(badSRs, badOpps, ERR_Handler.Context.RD);
+                ERR_Handler.getErrors(badSRs, badOpps);
+                //TODO: process result??!?!
```

Comment:
processInsertResults was not logging errors. It was only attempting to send an error notification, but this was not the right way to do it. It needs to be done after the rollback or it will be pulled out of the queue when we roll back.

This section is marked with "TODO", because I think we should do something with the errors (log them + send notifications). I just need to better understand why it was not being done before.